src/sql/workbench/contrib/notebook/browser/cellViews/codeCell.component.ts

```diff
@@ -32,13 +31,13 @@ export class CodeCellComponent extends CellView implements OnInit, OnChanges {
 		this._activeCellId = value;
 	}
 
-	@HostListener('document:keydown.escape', ['$event'])
-	handleKeyboardEvent() {
-		this.cellModel.active = false;
-		this._model.updateActiveCell(undefined);
+	// On click to edit code cell in notebook
+	@HostListener('click', ['$event']) onClick() {
+		this.toggleEditMode();
 	}
 
 	private _activeCellId: string;
+	private isEditMode: boolean;
```

Comment:
Why do we need to keep a separate state for this? Can't you just use the cellModel.isEditMode directly?